packages/axios-extension/src/abap/ui5-abap-repository-service.ts

```diff
@@ -54,6 +53,10 @@ export interface DeployConfig {
      * if set then the SafeMode url parameter will be set. SafeMode is by default active, to deactivate provide false
      */
     safeMode?: boolean;
+    /**
+     * if set to true, then additional info message will be logged
```

Comment:
I would update the comment to;

``
if set to true, then any additional info messages will be logged
```
Thus if the destination is not an onpremise destination and we wanted to logged additional info, this could be used as well.